chafa/internal/chafa-math-util.c

```diff
@@ -109,3 +109,9 @@ chafa_tuck_and_align (gint src_width, gint src_height,
 #endif
 }
 
+gint
+round_up_to_multiple_of (gint value, gint m)
```

Comment:
Nitpick: Non-static functions, including internal ones, should be namespaced (`chafa_`).

Reply:
Resolved in 6eb5a346f4c0690acdab7b3496a476fdc4e46e12.